### PR TITLE
Documentation pages

### PR DESCRIPTION
**Pull request by muddv**
Added a build script to generate documentation pages.
For pages to generate properly, please also see pull request to main PostCSS repository.

### CODE REVIEW

File: scripts/build-docs.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+  if (cls) attrs.class = cls
+  if (Array.isArray(body)) body = body.join('')
+  let attrsString = Object.keys(attrs)
+    .map(i => ` ${i}="${attrs[i]}"`)
+    .join('')
+  return `<${tagName}${attrsString}>${body}</${tagName}>`
+}
+
+function makeSidemenu(contents) {
+  let titles = contents.match(/(?<=<p><a(?:.*?)>)(.*?)(?=<)/gm).filter(title => title !== "").filter(title => title !== "PostCSS and Source Maps")
```
Comment (ai) [inline scripts/build-docs.js:141]: Can you format code by Prettier?

This line is definitely longer than we have in code standard

Comment (muddv) [reply scripts/build-docs.js:141]: Sure, will do now

Comment (muddv) [reply scripts/build-docs.js]: Formatted

**Comment by ai** [comment]
You need to fix CSS linter warnings.

**Comment by ai** [comment]
Can you also show a screenshot of any new page?

**Comment by muddv** [comment]
Some screenshots from random pages:
image
image
image
Last one is index if docs/ directory, maybe layout needs a bit of work.
Working CSS linter warnings now.

**Comment by ai** [comment]
> Last one is index if docs/ directory, maybe layout needs a bit of work.

We should not render `/README.md` here in `docs/`.

> Some screenshots from random pages

Text looks ugly. Seems like we have no normal styles with bullets for `<ul>`. Font for code examples are not much what we used on main page.

If you need a design inspiration, you can use main page styles and [Logux website](https://logux.io/) and [its styles](https://github.com/logux/logux.io/tree/main/src).

**Comment by muddv** [comment]
Okay, will get a different font for the code and make sure bullets will generate with <ul>

The font for code block is the same as in `api` pages (I think):
image
but I definitely can change it

Agree with not rendering `README.md` here, also will change that

Thank you for taking a look!

**Comment by ai** [comment]
> The font for code block is the same as in api pages

The problem is in inline `<code>`, not code blocks. Look at `Root` link in first screenshot.

**Comment by muddv** [comment]
Oh sorry, I see it now, need to change some selectors, because currently inline <code> does not inherit correct fonts

**Comment by ai** [comment]
Post new screenshots when you will be sure in doc pages design

**Comment by muddv** [comment]
More screenshots, I tweaked a few thing, should look better now.
image
image
image
image

**Pull request merged**